commands.js

```diff
@@ -2007,6 +2007,7 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (target === 'show') return this.parse('/showblacklist');
```

Comment:
Yeah I don't think this is the way to do it. Even if "show" doesn't come online ever, it's still pretty BP to do it this way.